space-cli/cmd/modules/operations/setup.go

```diff
@@ -135,7 +135,24 @@ func Setup(username, key, config, version, secret, clusterName string, dev bool,
 		"CLUSTER_ID=" + clusterID,
 		"PORT=" + portHTTPValue,
 	}
+	var runnerEnvs = []string{
+		"DEV=" + devMode,
+		"ARTIFACT_ADDR=store.space-cloud.svc.cluster.local:" + portHTTPValue, // TODO Change the default value in runner it starts with http
```

Comment:
Lets remove this

Reply:
what should i remove here ? Artifact addr